<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>1</string>
       </attribute>
       <widget class="QTextEdit" name="tab1_TextEdit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>481</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QTreeWidget" name="tab1_TreeWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>181</width>
          <height>381</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>C程序设计目录</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="tab1_Btn1">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>字体</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tab1_Btn2">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>颜色 </string>
        </property>
       </widget>
       <widget class="QPushButton" name="tab1_Btn3">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>背景</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>2</string>
       </attribute>
       <widget class="QTextEdit" name="tab2_TextEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>711</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="tab2_Btn1">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tab2_Btn2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tab2_Btn3">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>字体</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tab2_Btn4">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>颜色</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>3</string>
       </attribute>
       <widget class="QToolButton" name="tab3_Btn1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QToolButton" name="tab3_Btn2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QToolButton" name="tab3_Btn3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>O</string>
        </property>
       </widget>
       <widget class="QToolButton" name="tab3_Btn4">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>口</string>
        </property>
       </widget>
       <widget class="QFrame" name="drawingBoard">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>711</width>
          <height>391</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QToolButton" name="tab3_Btn5">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>存</string>
        </property>
       </widget>
       <widget class="QToolButton" name="tab3_Btn6">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>擦</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
